File: abc.com/src/main/java/application/specification/Specification.java
package application.specification;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import application.common.BasicInfo;

@Document
public class Specification extends BasicInfo{
	
	@Id
    private String id;
	private String specificationName;
	private String sectionId;
	private String sectionName;
	private String label;
	private String answer;
	
	
	public String getLabel() {
		return label;
	}

	public void setLabel(String label) {
		this.label = label;
	}

	public String getAnswer() {
		return answer;
	}

	public void setAnswer(String answer) {
		this.answer = answer;
	}

	public String getSectionName() {
		return sectionName;
	}

	public void setSectionName(String sectionName) {
		this.sectionName = sectionName;
	}

	public String getSectionId() {
		return sectionId;
	}

	public void setSectionId(String sectionId) {
		this.sectionId = sectionId;
	}

	public Specification(String specificationName, String label, String answer) {

		this.specificationName = specificationName;
		this.label = label;
		this.answer = answer ;
	}

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getSpecificationName() {
		return specificationName;
	}

	public void setSpecificationName(String specificationName) {
		this.specificationName = specificationName;
	}
	
	public Specification() {

		super();
	}

}

File: abc.com/client/src/app/specification/specification.ts
import { Class } from '@angular/core';

export class Specification 
{
    private  Id: string;
    private  specificationName: string;
    private  sectionId: string;
    private  label : string;
    private  answer : string;
    private  DOB:Date;
    private  sectionName:string;
     
    constructor() 
    {}
    public setId ( Id: string ) 
      {
      this.Id = Id;
      }
    public getId (): string 
      {
      return this.Id ;
      }
      public setDOB ( DOB: Date ) 
      {
      this.DOB = DOB;
      }
    public getDOB (): Date 
      {
      return this.DOB ;
      }
    
    public setSpecificationName ( specificationName: string ) 
    {
      this.specificationName = specificationName;
    }
    
    public getSpecificationName (): string 
    {
      return this.specificationName ;
    }

    public setSectionId ( sectionsId: string ) 
    {
      this.sectionId = sectionsId;
    }

    public getSectionId(): string 
    {
      return this.sectionId ;
    }
    public setAnswer (answer: string) 
    {
      return answer = answer ;
    }
    public getAnswer(): string 
    { 
      return this.answer ;
    }
    public setSectionName ( sectionsName: string ) 
    {
      this.sectionName = sectionsName;
    }

    public getSectionName(): string 
    {
      return this.sectionName ;
    }
    
    public setLabel (label: string) 
    {
      return label = label ;
    }
    public getLabel(): string 
    { 
      return this.label ;
    }
}